src/vtl.ts

```diff
@@ -29,6 +47,35 @@ export class VTL {
     this.statements.push(...statements);
   }
 
+  /**
+   * Name of a variable on the stash. Prefixes with the VTL engines stash name.
+   * @param name Name of the variable.
+   */
+  public abstract stashVariableName(name: string): string;
+  /**
+   * Evaluate an expression quietly i.e. without outputting to the VTL result.
+   * @param expr Expr to evaluate.
+   */
+  public abstract quiet(expr: string): void;
+  /**
+   * Return the value of the expression.
+   * @param expr Expr to evaluate and return.
+   */
+  public abstract return(expr: string | Expr): void;
+  /**
+   * Output a variable as JSON.
+   * @param reference Name of the variable to JSONify.
+   */
+  public abstract json(reference: string): string;
```

Comment:
These are Appsync only, right?

Reply:
Or is the plan to support them with different implementations? (assuming it can be done)

Reply:
There are equivalents. For `quiet` you can do `#set($discard = ...)`. For `return` just output the variable. This one probably won't be used/allowed for API GW, still figuring it all out. For `json` you use [The Beast](https://github.com/sam-goodwin/functionless/issues/23#issuecomment-1088336878)